markdown/htmlparser.py

```diff
@@ -200,3 +210,63 @@ def handle_pi(self, data):
     def unknown_decl(self, data):
         end = ']]>' if data.startswith('CDATA[') else ']>'
         self.handle_empty_tag('<![{}{}'.format(data, end), is_block=True)
+
+    # The rest has been copied from base class in standard lib to address #1036.
+    # As __startag_text is private, all references to it must be in this subclass.
+    # The last few lines of parse_starttag are reversed so that handle_starttag
+    # can override cdata_mode in certain situations (in a code span).
+        __starttag_text = None
```

Comment:
This line seems to have wrong indentation?

Reply:
Should this be defined in __init__?

Reply:
Yes, the indentation is wrong. It is not defined in `__init__` in the base class.